src/post/CreatePost.tsx

```diff
@@ -92,6 +94,7 @@ function CreatePost() {
         placeholder='제목을 입력하세요.'
       />
       <Div />
+      <Image src={previewImage ? previewImage : ''}></Image>
```

Comment:
ask;
previesImage 기본값이 빈문자열이니까 그냥
```suggestion
      <Image src={previewImage} />
```
이런 식으로는 안되나용?? 😄